src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml.cs

```diff
@@ -99,7 +101,7 @@ private async Task MigrateAsync()
         private void MigrationFinished()
         {
             migrationInProgress = false;
-            this.finishButton.Visibility = Visibility.Visible;
+            this.finishButton.IsEnabled = true;
```

Comment:
Ahh... thats how you grey it out haha, I knew it was possible but somehow didn't think of this

Reply:
Buttons (and other controls) can update their styles automatically based on the state of the button i.e. based on the value of various properties. Here, the button is changing its style based on the value of `IsEnabled`. Buttons appeared grey-ed out is the default styling behaviour, but we could change that if wanted.